features/promotion/applying_catalog_promotions/applying_all_available_types_catalog_promotions.feature

```diff
@@ -0,0 +1,35 @@
+@applying_catalog_promotions
+Feature: Applying all available catalog promotions
+    In order to be attracted to products
+    As a Visitor
+    I want to see discounted products in the catalog
+
+    Background:
+        Given the store operates on a channel named "Web-US" with hostname "web-us"
+        And the store operates on another channel named "Web-GB" with hostname "web-gb"
+        And the store has "Clothes" taxonomy
+        And the store has "Dishes" taxonomy
+        And the store has a "T-Shirt" configurable product
+        And this product main taxon should be "Clothes"
+        And this product has "PHP T-Shirt" variant priced at "$20.00" in "Web-US" channel
+        And "PHP T-Shirt" variant priced at "$30.00" in "Web-GB" channel
+        And the store has a "Mug" configurable product
+        And this product main taxon should be "Clothes"
+        And this product has "PHP T-Shirt" variant priced at "$5.00" in "Web-US" channel
+        And "PHP T-Shirt" variant priced at "$10.00" in "Web-GB" channel
+        And there is a catalog promotion "Clothes sale" that reduces price by "30%" and applies on "Clothes" taxonomy
+
+    @todo
+    Scenario: Applying multiple catalog promotions
+        And there is a catalog promotion "Winter sale" available in "Web-US" channel that reduces price by "30%" and applies on "PHP T-shirt" variant
+        When I change my current channel to "Web-US"
```

Comment:
do you need two Channels in those scenarios?

Reply:
We test channel CP. For better testing I can tested do product on second channel has proper price.

Reply:
IMO as you are using only one channel in these scenarios, there is no need for defining both od them